docker/armorblox/Pipfile

```diff
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+armorblox-sdk = "*"
```

Comment:
Didn't we say we have to use a specific version of the SDK?

Reply:
yeah Rajat, I think it was not updated by mistake. I will update it.

Reply:
Updated the armorblox-sdk new version in pipfile.